src/components/info/location-info.vue

```diff
@@ -0,0 +1,193 @@
+<script setup lang="ts">
+import { computed, ref, watch } from 'vue'
+import { useTranslation } from 'i18next-vue'
+import { storeToRefs } from 'pinia'
+import { useInfoStore } from '@/stores/info.store'
+import { Coordinate } from 'ol/coordinate'
+import useMap from '@/composables/map/map.composable'
+import {
+  getQRUrl,
+  getElevation,
+  getNearestAddress,
+  queryInfo,
+} from '@/services/info/location-info'
+
+import StreetView from '@/components/info/street-view.vue'
+
+const { t } = useTranslation()
+const { locationInfo } = storeToRefs(useInfoStore())
+
+const map = useMap().getOlMap()
+
+const shortUrl = ref()
+const qrUrl = ref()
+const elevation = ref()
+const address = ref()
+const clickCoordinateLuref = ref()
+const formatted_coordinates = ref({})
+
+// initial load of position infos if locationInfo is not initially undefined
+// async function, no need to await the completion, DOM will be updated via refs
+updateInfo(locationInfo.value)
+
+watch(locationInfo, updateInfo)
+
+async function updateInfo(location: Coordinate | undefined) {
+  if (location) {
+    const info = await queryInfo(location, map.getView().getProjection())
+    shortUrl.value = info.shortUrl
+    qrUrl.value = getQRUrl(info.shortUrl)
+    clickCoordinateLuref.value = info.clickCoordinateLuref
+    formatted_coordinates.value = info.formatted_coordinates
+    elevation.value = await getElevation(info.clickCoordinateLuref)
+    address.value = await getNearestAddress(info.clickCoordinateLuref)
```

Comment:
You are doing 3 await, this means the user is going to wait a bit more at each request. Please use parallele loading instead (eg. `Promise.allSettled`).

Reply:
yes, good idea, the awaits came together after refactoring,and I did not notice the possibility to regroup